pkg/cmd/repo/clone/clone.go

```diff
@@ -44,6 +44,9 @@ func NewCmdClone(f *cmdutil.Factory, runF func(*CloneOptions) error) *cobra.Comm
 			If the "OWNER/" portion of the "OWNER/REPO" repository argument is omitted, it
 			defaults to the name of the authenticating user.
 
+			If the repository is a fork, add the upstream repository as a remote,
```

Comment:
"add the upstream repository" might sound like an instruction that the user has to do, rather than an explanation of what the command will automatically do. How about:

> If the repository is a fork, an additional git remote named "upstream" will be added tracking the parent repository.

Reply:
Sorry, I missed this before you merged!